examples/IIM/ex9/BcData.h

```diff
@@ -0,0 +1,36 @@
+struct BcData
+{
+    const double U1;
+    const double U2;
+    
+    const double t_load;
+    const double tg_load;
+    
+//    static std::string inflow_data;
+    
+    const double wall;
+    const double d_in;
+    const double d_out;
+    const double h1_in;
+    const double h2_in;
+    const double h_out;
+    const double z_min;
+    const double z_max;
+    
+        BcData(Pointer<Database> input_db)

```

Comment:
We don't do a good job enforcing it but all our headers should compile on their own - i.e., we need the headers for `tbox::Pointer` and `tbox::Database` here.